src/main/java/org/fundacionjala/coding/jorge/FizzBuzz.java

```diff
@@ -0,0 +1,42 @@
+package org.fundacionjala.coding.jorge;
+
+import java.util.stream.IntStream;
+
+/**
+ *
+ */
+public final class FizzBuzz {
+
+    /**
+     * The first part.
+     *
+     * @param rango int[]
+     * @return Object[]
+     */
+    public Object[] fizzBuzz(final int[] rango) {
+        Object[] arrayInt = IntStream.rangeClosed(rango[0], rango[1]).mapToObj(FizzBuzz::changeFB1).toArray();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "arrayInt". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1488)